ext/lab2im/layers.py

```diff
@@ -1735,3 +1736,19 @@ def _single_blur(self, inputs):
 
     def compute_output_shape(self, input_shape):
         return input_shape
+
+
+def unpack_singleton(x):
```

Comment:
If x is a tuple or a list, then please annotate the argument (and the return) type in the function definition. See

https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions

Reply:
The reason I like type hints is that it greatly improves the ability of the IDE or a linter to spot errors in code that uses your functions. It can (at least partly) infer types and knows when you call your functions in the wrong way.

Reply:
:+1: Absolutely agree, type hints I like!